components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java

```diff
@@ -96,8 +96,8 @@
      */
     private String readPassiveSTSHtmlRedirectPage() {
         String redirectHtmlFilePath = CarbonUtils.getCarbonHome() + File.separator + "repository"
-                + File.separator + "resources" + File.separator + "security" + File.separator +
-                "sts_redirect.html";
+                + File.separator + "resources" + File.separator + "identity" + File.separator + "pages" + File.separator +
+                "sts_response.html";
```

Comment:
Better to rename sts_response.html as passivests_response.html, because "sts" implies the Active STS (though it doesn't use browser interactions).